tests/functional/test_locked_objects.py

```diff
@@ -9,223 +9,195 @@
+        prepared_cluster.host_add(host)
+        added_service = prepared_cluster.service_add(name="bookkeeper")
+    with allure.step("Run action: action locker"):
+        host.action_run(name="action-locker")
+    with allure.step("Check if host, cluster and service are locked"):
+        prepared_cluster.reread()
+        host.reread()
+        added_service.reread()
+        assert prepared_cluster.state == "locked"
+        assert host.state == "locked"
+        assert added_service.state == "locked"
+
+
+def test_run_service_action_locked_all_objects_in_cluster(
+    prepared_cluster: Cluster, host: Host
+):
+    with allure.step("Add host and service"):
+        prepared_cluster.host_add(host)
+        added_service = prepared_cluster.service_add(name="bookkeeper")
+    with allure.step("Run action: service lock"):
+        added_service.action_run(name="service-lock")
+    with allure.step("Check if host, cluster and service are locked"):
+        prepared_cluster.reread()
+        host.reread()
+        added_service.reread()
+        assert prepared_cluster.state == "locked"
+        assert host.state == "locked"
+        assert added_service.state == "locked"
+
+
+def test_cluster_should_be_unlocked_when_ansible_task_killed(prepared_cluster: Cluster):
+    with allure.step("Run cluster action: lock terminate"):
+        task = prepared_cluster.action_run(name="lock-terminate")
+    with allure.step("Check if cluster is locked and then terminate_failed"):
+        prepared_cluster.reread()
+        assert prepared_cluster.state == "locked"
+        task.wait()
+        prepared_cluster.reread()
+        assert prepared_cluster.state == "terminate_failed"
+
+
+def test_host_should_be_unlocked_when_ansible_task_killed(
+    prepared_cluster: Cluster, host: Host
+):
+    with allure.step("Add host"):
+        prepared_cluster.host_add(host)
+    with allure.step("Run action: lock terminate"):
+        task = prepared_cluster.action_run(name="lock-terminate")
+
+    with allure.step("Check if host is locked and then created"):
+        host.reread()
+        assert host.state == "locked"
+        task.wait()
+        host.reread()
+        assert host.state == "created"
+
+
+def test_service_should_be_unlocked_when_ansible_task_killed(prepared_cluster: Cluster):
+    with allure.step("Add service"):
+        added_service = prepared_cluster.service_add(name="bookkeeper")
+    with allure.step("Run action: lock-terminate"):
+        task = prepared_cluster.action_run(name="lock-terminate")
+    with allure.step("Check if service is locked and then created"):
+        added_service.reread()
+        assert added_service.state == "locked"
+        task.wait()
+        added_service.reread()
+        assert added_service.state == "created"
+
+
+def test_hostprovider_must_be_unlocked_when_his_task_finished(hostprovider: Provider):
+    with allure.step("Run action: action locker and create hostprovider"):
```

Comment:
Looks like typo in allure step
```suggestion
    with allure.step("Run action: action-locker for hostprovider"):
```

Reply:
fixed